src/modelconfig.hpp

```diff
@@ -116,6 +116,11 @@ class ModelConfig {
          */
     bool disabledCaching = false;
 
+    /**
+         * @brief Cache allowing parameter
```

Comment:
Isn't `disabledCaching` and `allowCache` too much of configuration? By that I mean this might be misleading to have 2 flags

Reply:
allowCache was added to be able to determine if such parameter was in config file. With disabledCaching we are not aware if `allow_cache` was set in config, therefore we could not determine if the model should be loaded or not.

Reply:
I suppose that it could work.